lib/jobs/ipmi-catalog.js

```diff
@@ -110,11 +110,26 @@ function ipmiCatalogJobFactory(
         assert.string(command);
         assert.object(obmSettings);
 
-        return [ 
+        // Some commands end with " || true"
+        // This tail needs to be deleted from the command args
+        // Otherwise the tail passed to ipmitool child process will cause error response
+        // this tail is stored and used to determine whether continue sending other commands
+        var ignore = false;
+        var tailIdx = command.indexOf(' ||');
```

Comment:
Thanks @brianparry . After investigation, it is found that "|| true" is not the best solution here, since tasks already uses acceptedResponseCodes as a flag. Will close this PR and open another for a better structure.